types/json2xls/index.d.ts

```diff
@@ -0,0 +1,32 @@
+type Json = string | number | boolean | { [key: string]: Json } | Json[];
```

Comment:
'export' is to export, export default is 'export default', there is no magic to access not exported declarations IMO (should not work).
try just simple (expressing intent of the exported module):
```ts
import express = require("express");

declare function json2xls(json: any, options?: json2xls.Config): Buffer;

declare namespace json2xls {
    interface Config {
        fields?: Record<string, string> | string[] | undefined;
        style?: string | undefined;
    }

    function middleware(
        req: express.Request,
        res: express.Response,
        next: express.NextFunction,
    ): void;
}

export = json2xls;
```

Reply:
It is a smaller and nicer declaration however **any** type in json2xls function **json** key might be little bit wide. Because of the face, It is real Json text notation? I shouldn't put any object type to this function?

I should keep Json maybe?

Reply:
Btw I wrote the declaration file from tests folder of the main package

Reply:
Nevermind, I wont try to map all serializable types for JSON 😄